File: casadaApi/src/main/java/com/casada/casadaApi/entities/Massagechairclass.java
package com.casada.casadaApi.entities;

import lombok.Data;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import java.io.Serializable;

/**
 * $table.getTableComment()
 */
@Data
@Entity
@Table(name = "massagechairclass")
public class Massagechairclass implements Serializable {

    private static final long serialVersionUID = 1L;

    @Id
    @Column(name = "massagechairclassid", nullable = false)
    private Long massagechairclassid;

    @Column(name = "massagechairclassname", nullable = false)
    private String massagechairclassname;

}

File: casadaGUI/casada/lib/data/massage_chair_class.dart
class MassageChairClass {
  int? massageChairClassId;
  String? massageChairClassName;

  MassageChairClass({this.massageChairClassId, this.massageChairClassName});

  MassageChairClass.fromJson(Map<String, dynamic> json) {
    massageChairClassId = json['massageChairClassId'];
    massageChairClassName = json['massageChairClassName'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['massageChairClassId'] = this.massageChairClassId;
    data['massageChairClassName'] = this.massageChairClassName;
    return data;
  }
}